editor/editor_themes.cpp

```diff
@@ -342,7 +350,13 @@ void editor_register_and_generate_icons(Ref<Theme> p_theme, bool p_dark_theme =
 		for (int i = 0; i < editor_bg_thumbs_count; i++) {
 			const int index = editor_bg_thumbs_indices[i];
 			const int is_exception = exceptions.has(editor_icons_names[index]);
-			const Ref<ImageTexture> icon = editor_generate_icon(index, !p_dark_theme && !is_exception, scale, force_filter, dark_icon_color_dictionary);
+
+			Ref<ImageTexture> icon;
+			if (!p_dark_theme && !is_exception) {
+				icon = editor_generate_icon(index, scale, force_filter, icon_color_map);
+			} else {
+				icon = editor_generate_icon(index, scale, force_filter, dummy_color_map);
```

Comment:
```suggestion
				icon = editor_generate_icon(index, scale, force_filter, HashMap<Color, Color>());
```

Or maybe just set `HashMap<Color, Color>()` as a default argument value.

Reply:
Neither of the two options compiled on all platforms, which is why I went this way.